src/lua-runtime-fastpm.lua

```diff
@@ -59,7 +59,9 @@ schema.ncdm_sphere_scheme.choices = {
 }
 schema.declare{name='ncdm_matterlike', type='boolean', required=false, default=true, help="Approximate ncdm as matter-like in the background? If true, Omega_ncdm~1/a^3."}
 schema.declare{name='ncdm_freestreaming', type='boolean', required=false, default=true, help="Treat ncdm as free-streaming? If true, source terms ~Omega_c; if false, ~Omega_m."}
-
+schema.declare{name='ncdm_linearresponse', type='boolean', required=false, default=false, help="Enable the linear response module for the neutrino contribution. Adds neutrinos to the code without needing extra particles."}
+schema.declare{name='ncdm_timetransfer', type='number', required=false, default=0.01, help="Redshift of the neutrino transfer function."}
```

Comment:
Is ncdm_timetransfer consistent with name of argument for the time of input matter power?

Reply:
Looks like the other parameters are like ….redshift

Reply:
Maybe I should just enforce that it uses the same parameter as for the input matter power?

Reply:
See: [b0f8446](https://github.com/fastpm/fastpm/pull/114/commits/b0f8446c26bd2fb747cc975a4539334df54cd590)

Reply:
Keeping it as a separate parameter then you can make them identical in the lua file. What about call it ncdm_transfer_redshift (to echo linear_density_redshift)?

Reply:
Ok! Done.